webapp/src/monaco.tsx

```diff
@@ -338,6 +338,27 @@ export class Editor extends srceditor.Editor {
             }
         })
 
+        if (pxt.appTarget.appTheme.hasReferenceDocs) {
+            let referenceContextKey = this.editor.createContextKey("editorHasReference", false)
+            this.editor.addAction({
+                id: "reference",
+                label: lf("Show Definition"),
```

Comment:
Yeah. I was considering that. My only issue with just Help, is it doesn't portray the message that you you need to "Right Click" on a specific method or namespace to get the reference doc for it. (ie: you can't just click anywhere in the window and go to "Help")
"Help with function" or something along those lines might be good.